ui/src/containers/NodePageMetricsTab.js

```diff
@@ -526,21 +527,23 @@ const NodePageMetricsTab = ({
           {showAvg && !avgStatsData['cpuUsage'].length ? <Loader /> : null}
         </MetricsToggleWrapper>
         {api && api.url_grafana && (
-          <Button
-            text={intl.formatMessage({ id: 'advanced_metrics' })}
-            variant={'buttonSecondary'}
-            icon={<i className="fas fa-external-link-alt" />}
-            size={'small'}
+          <a
             href={`${api.url_grafana}/dashboard/db/nodes-detailed?var-DS_PROMETHEUS=Prometheus&var-job=node-exporter&var-name=${hostnameLabel}`}
             target="_blank"
             rel="noopener noreferrer"
             data-cy="advanced_metrics_node_detailed"
-          />
+          >
+            <Button
+              label={intl.formatMessage({ id: 'advanced_metrics' })}
+              variant={'secondary'}
+              icon={<i className="fas fa-external-link-alt" />}
+            />
+          </a>
         )}
         <Dropdown
           items={metricsTimeSpanDropdownItems}
           text={metricsTimeSpan}
-          size="small"
+          size="large"
```

Comment:
It's not related to this PR.
I understand here you want to make sure the button and the dropdown have the same size.
IMO, the issue will be handled by select/dropdown component, anyways, manually set the size of the dropdown won't solve the issue for all the screens. 
hence I suggest we remove it.